### PR TITLE
Add Evmos Network

### PR DESCRIPTION
**Pull request by NoahMarconi**
Changes proposed in this pull request:

Add evmos network ([https://chainlist.org/chain/9001](https://chainlist.org/chain/9001))

### CODE REVIEW

**Review by ChaituVR — CHANGES_REQUESTED** [review]
@NoahMarconi Looks like none of these RPC nodes are full archive nodes, we need to full archive node to be able to support

**Comment by ChaituVR** [comment]
Also need to verify and publish code here https://escan.live/address/0x2764b262e60d3ac96999f7e73b0628b09cbb367e#contract

**Comment by NoahMarconi** [comment]
I've now updated with an archival rpc provided by the evmos team and linked to thier blockscout explorer.

**Comment by NoahMarconi** [comment]
@ChaituVR evmos informed me no public archival nodes will exist prior to the restart. POST restart, the above should do it.

What blockheight do you need?

File: src/networks.json [hunk 1]
```diff
@@ -1880,5 +1880,23 @@
...
     "start": 43317,
     "logo": "ipfs://QmXvum7SNVaAqAc2jNzR1NpNZN1GGvNaKWydg8a1GEDQ7y"
+  },
+  "9001": {
+    "key": "9001",
+    "name": "Evmos Mainnet",
+    "shortName": "EVMOS",
+    "chainId": 9001,
+    "network": "mainnet",
+    "multicall": "0x2764b262E60D3Ac96999F7e73b0628B09CbB367E",
```
Comment (ChaituVR) [inline src/networks.json:1890]: https://blockscout.evmos.org/address/0x2764b262E60D3Ac96999F7e73b0628B09CbB367E/contracts#address-tabs Can verify and publish code here?

Comment (ChaituVR) [reply src/networks.json:1890]: >  evmos informed me no public archival nodes will exist prior to the restart. POST restart, the above should do it.
What blockheight do you need?

@NoahMarconi It should atleast work from multicall contract creation block, i.e 59316 in this case

File: src/networks.json [hunk 2]
```diff
@@ -1880,5 +1880,23 @@
...
+    "rpc": [
+      "https://eth.bd.evmos.org:8545",
+      "https://evmos-mainnet.public.blastapi.io",
+      "https://evmos-json-rpc.stakely.io"
+    ],
+    "explorer": {
+      "url": "http://blockscout.evmos.org"
+    },
+    "start": 8791997,
```
Comment (ChaituVR) [inline src/networks.json:1899]:
```suggestion
    "start": 59316,
```

**Comment by ChaituVR** [comment]
Closing this as stale, feel free to reopen if needed

**Pull request closed without merging**